src/vcpkg/commands.build.cpp

```diff
@@ -1690,30 +1690,110 @@ namespace vcpkg
         return "https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+" + spec_name;
     }
 
-    static std::string make_gh_issue_open_url(StringView spec_name, StringView triplet, StringView path)
+    static std::string make_gh_issue_open_url(StringView spec_name, StringView triplet, StringView body)
     {
         return Strings::concat("https://github.com/microsoft/vcpkg/issues/new?title=[",
                                spec_name,
                                "]+Build+error+on+",
                                triplet,
-                               "&body=Copy+issue+body+from+",
-                               Strings::percent_encode(path));
+                               "&body=",
+                               Strings::percent_encode(body));
+    }
+
+    enum class CIType
```

Comment:
I think we already have something like this floating around for telemetry purposes and command line settings to control whether it happens; I believe we should reuse that if possible.

Reply:
See https://github.com/microsoft/vcpkg-tool/blob/b733d994a01e4e0392dde2d6f1adc75bb9561809/src/vcpkg/vcpkgcmdarguments.cpp#L18-L67